src/styles/_catalog.scss

```
@@ -14,9 +15,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all ease $t-duration;
```

avoid using `all` - better specify particular ones
```suggestion
    transition: transform ease $t-duration;
```